lumigator/backend/backend/services/datasets.py

```diff
@@ -145,7 +145,9 @@ def _save_dataset_to_s3(self, temp_fname, record):
             dataset_key = self._get_s3_key(record.id, record.filename)
             dataset_path = self._get_s3_path(dataset_key)
             # Deprecated!!!
-            dataset_hf.save_to_disk(dataset_path, fs=self.s3_filesystem)
+            dataset_hf.save_to_disk(
+                dataset_path, storage_options=self.s3_filesystem.storage_options
+            )
```

Comment:
This fixes a HF dataset deprecation warning, the HF warning message recommended this as the way to fix it